src/shared_utilities/Domains.jl

```diff
@@ -196,9 +198,11 @@ struct Plane{FT} <: AbstractDomain{FT}
     xlim::Tuple{FT, FT}
     "Domain interval limits along y axis, in meters"
     ylim::Tuple{FT, FT}
+    "When not `nothing`, a Tuple that contains the center `long` and `lat`."
+    longlat::Union{Nothing, Tuple{FT, FT}}
     "Number of elements to discretize interval, (nx, ny)"
     nelements::Tuple{Int, Int}
-    "Flags for periodic boundaries; only true is supported"
```

Comment:
likewise here

Reply:
I added a note saying that only "no lateral flow is supported"